recipes/modelx/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "modelx" %}
+{% set version = "0.18.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/modelx-{{ version }}.tar.gz
+  sha256: c1d23c54d3a9f3a84c1dbc500a53236059cf4310934d66cfd545b28c15a79050
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - asttokens
+    - networkx >=2.0
+    - python >=3.6
+
+test:
+  imports:
+    - modelx
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/fumitoh/modelx
+  summary: Numerical computing tool to use Python like a spreadsheet
+  license: LGPL-3.0-only
+  license_file: LICENSE.txt
```

Comment:
Use LICENSE.LESSER.txt which is included in the package (it's the one corresponding to LGPL).

Reply:
baee2f14f9ba8c04ea2c3bd6022ee97c00b09244 fixes the file name.